tensorflow/compiler/xla/service/llvm_ir/llvm_util.cc

```diff
@@ -93,6 +93,20 @@ llvm::CallInst* EmitCallToIntrinsic(
   return b->CreateCall(intrinsic, AsArrayRef(operands));
 }
 
+llvm::CallInst* EmitCallToTargetIntrinsic(
+    TargetIntrinsicEnum intrinsic_en, absl::Span<llvm::Value* const> operands,
+    absl::Span<llvm::Type* const> overloaded_types,
+    LLVMTargetIRBuilder& llvm_target_ir_builder) {
+  llvm::IRBuilder<>* b = llvm_target_ir_builder.builder();
+  llvm::Module* module = ModuleFromIRBuilder(b);
+  llvm::Intrinsic::ID intrinsic_id =
+      llvm_target_ir_builder.GetTargetMachineFeatures()->GetIntrinsicID(
+          intrinsic_en);
```

Comment:
`en` is not a common abbreviation, it needs to be changed.  Simply `intrinsic` would be fine, maybe you call the other one `intrinsic_id` as you do, or maybe you call it `llvm_intrinsic` or something.